src/tests/suite/daos_rebuild.c

```diff
@@ -1338,8 +1338,13 @@ rebuild_kill_PS_leader_during_rebuild(void **state)
 	}
 	rebuild_io(arg, oids, OBJ_NR);
 
-	daos_kill_server(arg, arg->pool.pool_uuid, arg->group,
-			 arg->pool.alive_svc, 6);
+	/* kill non-leader rank */
+	if (leader != 6)
+		daos_kill_server(arg, arg->pool.pool_uuid, arg->group,
+				 arg->pool.alive_svc, 6);
+	else
+		daos_kill_server(arg, arg->pool.pool_uuid, arg->group,
+				 arg->pool.alive_svc, 5);
```

Comment:
The name of this test is `rebuild_kill_PS_leader_during_rebuild` so is this even valid?

Reply:
The test is supposed to kill PS leader during rebuild,   this is the first kill to trigger the rebuild,  then during the rebuild, the 2nd kill will kill the ps leader.